pkg/stacks/templates/engines/helm2.go

```diff
@@ -72,16 +71,16 @@ func (her *Helm2EngineRunner) CreateConfig(claim *unstructured.Unstructured, hc
 	s, ok := claim.Object[spec]
 
 	if !ok {
-		her.Log.V(logging.Debug).Info("Spec not found on claim; not creating engine configuration", "claim", claim)
+		her.Log.Debug("Spec not found on claim; not creating engine configuration", "claim", claim)
 	}
 
-	her.Log.V(logging.Debug).Info("Converting configuration", "spec", s)
+	her.Log.Debug("Converting configuration", "spec", s)
 	configContents, err := yaml.Marshal(s)
 
-	her.Log.V(logging.Debug).Info("Configuration contents as yaml", "configContents", configContents)
+	her.Log.Debug("Configuration contents as yaml", "configContents", configContents)
 
 	if err != nil {
-		her.Log.Error(err, "Error marshaling claim spec as yaml!", "claim", claim)
+		her.Log.Info("Error marshaling claim spec as yaml!", "claim", claim, "error", err)
```

Comment:
I agree with downgrading the level of this message. In fact, I believe we'd be okay with it at `Debug`, since the caller in the render phase controller also logs any returned error. I see that line seems to also log the error at `Info` - what are we thinking for the guideline for when to log at `Error`?

Reply:
Never - https://godoc.org/github.com/crossplaneio/crossplane-runtime/pkg/logging#Logger doesn't have an `Error` method. :) There are links to articles describing this philosophy in the package doc. Instead the convention is to include `"error", err` in the structured log arguments.